src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--wed calendar--month-length--30">
```

Comment:
Consider the first sentence of the checklist 
Display a calendar in the middle of the screen
To add external stylling to center it you might need to introduce one more block like `page`

Reply:
Horizontally centered.